components/Paragraph/Paragraph.css

```diff
@@ -1,56 +1,29 @@
 .msrd .msrd-Paragraph {
-  font-size: 1rem;
+  font: var(--font-03);
+  letter-spacing: var(--font-03-letter-spacing);
   font-weight: normal;
   margin: 0;
 }
 
-.msrd .msrd-Paragraph--01 {
-  font: var(--font-01);
-  letter-spacing: var(--font-01-letter-spacing);
+@media (min-width: 48em) {
+  .msrd .msrd-Paragraph {
+    font: var(--font-04);
+    letter-spacing: var(--font-04-letter-spacing);
+  }
 }
 
-.msrd .msrd-Paragraph--02 {
+.msrd .msrd-Paragraph--small {
   font: var(--font-02);
   letter-spacing: var(--font-02-letter-spacing);
 }
 
-.msrd .msrd-Paragraph--03 {
-  font: var(--font-03);
-  letter-spacing: var(--font-03-letter-spacing);
-}
-
-.msrd .msrd-Paragraph--04 {
+.msrd .msrd-Paragraph--large {
   font: var(--font-04);
-  letter-spacing: var(--font-04-letter-spacing);
-}
-
-.msrd .msrd-Paragraph--05 {
-  font: var(--font-05);
-  letter-spacing: var(--font-05-letter-spacing);
+  letter-spacing: var(--font-02-letter-spacing);
```

Comment:
You've paired `font-04` with `font-02` letter spacing. Deliberate?

Reply:
Nope

Reply:
Nope